diff --git a/VintagestoryLib/Properties/AssemblyInfo.cs b/VintagestoryLib/Properties/AssemblyInfo.cs
--- a/VintagestoryLib/Properties/AssemblyInfo.cs
+++ b/VintagestoryLib/Properties/AssemblyInfo.cs
@@ -14,4 +14,4 @@
 [assembly: ComVisible(false)]
 [assembly: Guid("8495b9b8-6974-45de-b896-87f195304ede")]
 [assembly: AssemblyFileVersion("1.22.7")]
-[assembly: AssemblyVersion("1.22.7.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
